torch_xla/debug/metrics.py

```diff
@@ -57,3 +57,20 @@ def clear_metrics():
 def metrics_report():
   """Retrieves a string containing the full metrics and counters report."""
   return torch_xla._XLAC._xla_metrics_report()
+
+
+def short_metrics_report(counter_names=None, metric_names=None):
+  """Retrieves a string containing the full metrics and counters report.
+
+  Args:
+    counter_names (list): The list of counter names whose data needs to be printed.
+    metric_names (list): The list of metric names whose data needs to be printed.
+  """
+  if counter_names == None:
```

Comment:
`if not counter_names` or `if counter_names is not None` would be more idiomatic here if you don't want to make this list a default argument.